### PR TITLE
Add Basic Mathematical Tools for Imaging & Visualization

### PR DESCRIPTION
**Pull request by kordianbruck**
- [X] I made sure that the arrays in `routes.php` are still alphabetically sorted. :sparkles:

### CODE REVIEW

**Comment by kordianbruck** [comment]
@Husain0007 can you update your fork please?

**Comment by Husain0007** [comment]
@kordianbruck I'm unable to understand what you mean.
I have forked the repo from TUM-Devs, should I fork from somewhere else?

**Comment by joschahenningsen** [comment]
Hi @Husain0007 :wave: thanks for contributing!
Your fork is out of date with this original repo because there are new changes since you forked it. [this](https://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository) is a great resource if you want to understand how to update your fork.

If that doesn't work for you, you can also delete your fork and fork the repository again, make your changes and open a new PR.

**Comment by Husain0007** [comment]
@joschahenningsen thanks for sharing that link.
I executed the commands and got a merge conflict in "I2DL Routes Update".
Screenshot 2021-01-07 at 8 26 16 PM.
Can you advise how to resolve this?

**Comment by joschahenningsen** [comment]
I think it would be easier to create a fresh fork, but it's always useful to learn something new :)

Your file should now look something like this: 
```
<<<<<<< HEAD
        'ibmi'             => [
=======
        'i2dl' => [
            'description' => 'Introduction to Deep Learning',
            'target'      => 'https://niessner.github.io/I2DL/',
        ],
        'ibmi' => [
>>>>>>> ac56234... I2DL Routes Update
```

You'll have to update it such that it contains your changes and lines with <<<<<<<, >>>>>>> or ======= are gone. Then you can run 
```
git add redirect/routes.php
git rebase --continue
```
This process may be repeated for multiple commits. After you are done you can push your changes.

**Comment by Husain0007** [comment]
@joschahenningsen  thank you so much for clearing that up  :) 
I have made the requisite changes and my branch is now up to date.
If some issues still persist, please let me know.

**Review by joschahenningsen — CHANGES_REQUESTED** [review]
I'm glad that I could help! Could you please remove `.DS_Store` as it has nothing to do with this website. Also, did you want to add i2dl as well? That change is not present now.

File: redirect/routes.php [hunk 1]
```diff
@@ -261,7 +265,7 @@ class Route {
             'description' => 'HyPer DB Webschnittstelle',
             'target'      => 'http://hyper-db.com/interface.html',
         ],
-        'ibmi'             => [
+        'ibmi' => [
```
Comment (joschahenningsen) [inline redirect/routes.php:268]: This still could be formatted, then I'm ready to merge :tada:

**Review by joschahenningsen — APPROVED** [review]

**Comment by joschahenningsen** [comment]
Awesome, thanks for contributing! :sparkles: If you want to learn more about git, git-tut.tum.sexy contains a very fun tutorial!

**Pull request merged**